sagemaker_neo_compilation_jobs/deploy_huggingface_model_on_Inf1_instance/inf1_bert_compile_and_deploy.ipynb

```diff
@@ -0,0 +1,1006 @@
+{
```

Comment:
<p>Clarity nit: why is it important to set the <code>return_dict</code> parameter to <code>False</code> when instantiating the model?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Please add some details on what is the default value of this <code>return_dict</code>  parameter and what will happen if it is set to<code>True</code>.  </p>